### ISSUE TITLE
NameError: name 'libname' is not defined

### ISSUE THREAD

**Issue by kozo2** [pypa/auditwheel#30]
I applied auditwheel to a project named E-Cell.
But there was NameError for this.
If you have any suggestions in this regard, please let me know.

```
[root@db4ffc7bd536 python]# auditwheel show dist/ecell-4.0.0-cp27-cp27mu-linux_x86_64.whl 

ecell-4.0.0-cp27-cp27mu-linux_x86_64.whl is consistent with the
following platform tag: "linux_x86_64".

The wheel references the following external versioned symbols in
system-provided shared libraries: CXXABI_1.3.1, GLIBC_2.4,
GLIBCXX_3.4, GCC_3.0.

The following external shared libraries are required by the wheel:
{
    "libc.so.6": "/lib64/libc-2.5.so",
    "libgcc_s.so.1": "/lib64/libgcc_s-4.1.2-20080825.so.1",
    "libgsl.so.0": "/usr/lib64/libgsl.so.0.14.0",
    "libgslcblas.so.0": "/usr/lib64/libgslcblas.so.0.0.0",
    "libhdf5.so.9": null,
    "libhdf5_cpp.so.9": null,
    "libm.so.6": "/lib64/libm-2.5.so",
    "libpthread.so.0": "/lib64/libpthread-2.5.so",
    "libstdc++.so.6": "/usr/lib64/libstdc++.so.6.0.8"
}

In order to achieve the tag platform tag "manylinux1_x86_64" the
following shared library dependencies will need to be eliminated:

libgsl.so.0, libgslcblas.so.0, libhdf5.so.9, libhdf5_cpp.so.9
[root@db4ffc7bd536 python]# auditwheel repair dist/ecell-4.0.0-cp27-cp27mu-linux_x86_64.whl 
Repairing ecell-4.0.0-cp27-cp27mu-linux_x86_64.whl
Traceback (most recent call last):
  File "/usr/local/bin/auditwheel", line 11, in <module>
    sys.exit(main())
  File "/opt/_internal/cpython-3.5.1/lib/python3.5/site-packages/auditwheel/main.py", line 49, in main
    rval = args.func(args, p)
  File "/opt/_internal/cpython-3.5.1/lib/python3.5/site-packages/auditwheel/main_repair.py", line 81, in execute
    update_tags=args.UPDATE_TAGS)
  File "/opt/_internal/cpython-3.5.1/lib/python3.5/site-packages/auditwheel/repair.py", line 79, in repair_wheel
    libname)
NameError: name 'libname' is not defined

```

**Comment by rmcgibbo** [comment]
Fixed in #31. You still have a problem that libhdf5.so.9 and libhdf5_cpp.so.9 were not found. The NameError just occurred when trying to report this as an exception.

**Comment by kozo2** [comment]
Thank you so much for your help.
I replaced ```libname``` to ```soname```, and copied libhdf5*.so.9 files to ```/usr/lib64```.
Now auditwheel works well.

```
[root@db4ffc7bd536 python]# auditwheel repair dist/ecell-4.0.0-cp27-cp27mu-linux_x86_64.whl 
Repairing ecell-4.0.0-cp27-cp27mu-linux_x86_64.whl
Grafting: /usr/lib64/libsz.so.2 -> ecell4/.libs/libsz-d3a5fd2a.so.2
Grafting: /usr/lib64/libgsl.so.0.14.0 -> ecell4/.libs/libgsl-b885fd68.so.0.14.0
Grafting: /usr/lib64/libhdf5.so.9 -> ecell4/.libs/libhdf5-0433ec10.so.9
Grafting: /usr/lib64/libhdf5_cpp.so.9 -> ecell4/.libs/libhdf5_cpp-f1964f1f.so.9
Grafting: /usr/lib64/libgslcblas.so.0.0.0 -> ecell4/.libs/libgslcblas-15b2d0fe.so.0.0.0
Grafting: /lib64/libz.so.1.2.3 -> ecell4/.libs/libz-a147dcb0.so.1.2.3
Setting RPATH: ecell4/bd.so to "$ORIGIN/.libs"
Setting RPATH: ecell4/egfrd.so to "$ORIGIN/.libs"
Setting RPATH: ecell4/core.so to "$ORIGIN/.libs"
Setting RPATH: ecell4/gillespie.so to "$ORIGIN/.libs"
Setting RPATH: ecell4/spatiocyte.so to "$ORIGIN/.libs"
Setting RPATH: ecell4/ode.so to "$ORIGIN/.libs"
Setting RPATH: ecell4/meso.so to "$ORIGIN/.libs"
Previous filename tags: linux_x86_64
New filename tags: manylinux1_x86_64
Previous WHEEL info tags: cp27-cp27mu-linux_x86_64
New WHEEL info tags: cp27-cp27mu-manylinux1_x86_64

Fixed-up wheel written to /root/ecell4/python/wheelhouse/ecell-4.0.0-cp27-cp27mu-manylinux1_x86_64.whl
```
